docs/lib/analytics/fragments/js/getting-started.md

```diff
@@ -1,3 +1,36 @@
+The Analytics category enables you to collect analytics data for your App. The Analytics category comes with built-in support for [Amazon Pinpoint](https://aws.amazon.com/pinpoint) and [Amazon Kinesis](https://aws.amazon.com/kinesis) (Kinesis support is currently only available in the Amplify JavaScript library).
+
+## Goal
+
+To setup and configure your application with Amplify Analytics and record an analytics event.
+
+## Prerequisites
+* [Install and configure Amplify CLI](https://docs.amplify.aws/cli/start/install)
+
+## Set up Analytics backend
+
+Run the following command in your project's root folder. The CLI will prompt configuration options for the Analytics category such as Amazon Pinpoint resource name and analytics event settings.
+
+> The Analytics category utilizes the Authentication category behind the scenes to authorize your app to send analytics events.
+
+```bash
+amplify add analytics
+```
+
+```console
+? Provide your pinpoint resource name: 
+    `yourPinpointResourceName`
+Adding analytics would add the Auth category to the project if not already added.
+? Apps need authorization to send analytics events. Do you want to allow guests and unauthenticated users to send analytics events? (we recommend you allow this when getting started) 
+    `Yes`
+```
+
+To deploy your backend, run:
+
+```bash
+amplify push
+```
+
```

Comment:
wow JS doc is long, please add this section back that was removed earlier:

## View analytics console

From the terminal run the following command. Navigate to the *Analytics* tab, and then choose *View in Pinpoint*.

```console
amplify console
```

Reply:
Added